app/serializable/serializable_submission.rb

```diff
@@ -2,6 +2,7 @@ class SerializableSubmission < JSONAPI::Serializable::Resource
   type 'submissions'
 
   has_many :entries
+  has_many :files
```

Comment:
This reapplies https://github.com/dxw/DataSubmissionServiceAPI/pull/9/commits/fe2eb4352013feccaa379c084b16fc5e8537ed7e - it may be that the branch needs rebasing against develop.